WordPress/src/main/res/layout/main_activity.xml

```diff
@@ -1,18 +1,24 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- used to host quick start focus view - must be RelativeLayout or FrameLayout -->
+<?xml version="1.0" encoding="utf-8"?><!-- used to host quick start focus view - must be RelativeLayout or FrameLayout -->
```

Comment:
Minor (🔍): This comment ended up being on the same line as the `<?xml ... ?>` definition due to the automatic refactoring. Consider moving it back to the below line.

Reply:
Ohh yesss! Moved back in 43d927d.

Reply:
Awesome! 🙏